### PR TITLE
Aniview plugin

### PR DESCRIPTION
**Pull request by DispatchCommit**

### CODE REVIEW

**Comment by DispatchCommit** [comment]
This needs to be rebased

**Comment by jessopb** [comment]
There was some disagreement whether changing the behavior of auto-play/muted was ok. 
Xander suggested making an issue to fix autoplay muted on embeds as an 'acceptable regression'.
Will merge soon.

File: ui/component/viewers/videoViewer/internal/videojs.jsx [hunk 1]
```diff
@@ -13,8 +13,27 @@ import hlsQualitySelector from './plugins/videojs-hls-quality-selector/plugin';
 import recsys from './plugins/videojs-recsys/plugin';
 import qualityLevels from 'videojs-contrib-quality-levels';
 import isUserTyping from 'util/detect-typing';
+import 'videojs-contrib-ads';
+import 'videojs-ima';
+// import aniview from './plugins/videojs-aniview/plugin';
```
Comment (jessopb) [inline ui/component/viewers/videoViewer/internal/videojs.jsx:18]: why is this here / commented out?

File: ui/component/viewers/videoViewer/internal/videojs.jsx [hunk 2]
```diff
@@ -517,13 +574,45 @@ export default React.memo<Props>(function VideoJs(props: Props) {
...
       // I think this is a callback function
       onPlayerReady(player);
     });
 
+    // pre-roll ads
+    // This must be initialized earlier than everything else
+    // otherwise a race condition occurs if we place this in the onReady call back
+    if (allowPreRoll && !SIMPLE_SITE && window.google) {
+      const google = window.google;
+      // player.aniview();
```
Comment (jessopb) [inline ui/component/viewers/videoViewer/internal/videojs.jsx:608]: We could just do IS_WEB to allow preroll...

Comment (jessopb) [reply ui/component/viewers/videoViewer/internal/videojs.jsx:608]: Also, // player.aniview commented out?

**Review by jessopb — COMMENTED** [review]
Confused about a few things here.

File: ui/component/viewers/videoViewer/internal/videojs.jsx (hunk 1 above)
Comment (jessopb) [reply ui/component/viewers/videoViewer/internal/videojs.jsx:18]: // @if TARGET="web" 
is a possible way to merge to master.
